validator/src/main.rs

```diff
@@ -2308,10 +2294,6 @@ pub fn main() {
 
     let private_rpc = matches.is_present("private_rpc");
     let do_port_check = !matches.is_present("no_port_check");
-    let no_rocksdb_compaction = true;
```

Comment:
Originally I was trying to find out whether we have arguments to change this, but I didn't see any related arguments.  This statement would be a confirmation that we have already hard-coded to have no manual compaction.

Reply:
Correct, and for the sake of paper trail, here is the PR where this value was disconnected from validator arg and became constant:
https://github.com/solana-labs/solana/pull/16697/files#diff-59e2068dd4d49c21ab87f3a5d8792d2daca826ab6924b32413e551c0e3035f61L2019-R2019